__tests__/extension.test.js

```diff
@@ -0,0 +1,44 @@
+const hasElement = async (page, selector, timeout) => {
+  try {
+    const element = await page.waitForSelector(selector, { timeout })
+    return Boolean(element)
+  } catch (error) {
+    return false
+  }
+}
+
+describe(`Extension visibility`, () => {
```

Comment:
This is an example file to demo how to test basic features and get the screenshots.